.github/workflows/e2e_custom_cl.yml

```diff
@@ -73,23 +73,34 @@ jobs:
     permissions:
       id-token: write
       contents: read
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
+      - name: Get core ref from PR body
```

Comment:
Should we move this out to its own job with an output and then use that output in both places instead of duplicated the shell script?